### DISCUSSION TITLE
Error when upgrade cask

### DISCUSSION THREAD

**Discussion by alxrey** [Homebrew/discussions#3378] [Casks]
Each time I upgrade [KiCAD](https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/kicad.rb) I get this error :
```
brew upgrade kicad 
==> Upgrading 1 outdated package:
kicad 6.0.5-0 -> 6.0.6-1
==> Upgrading kicad
==> Downloading https://kicad-downloads.s3.cern.ch/osx/stable/kicad-unified-6.0.
######################################################################## 100.0%
==> Removing files:
/Applications/KiCad
Password:
/Library/Application Support/kicad/demos
==> Purging files for version 6.0.6-1 of Cask kicad
Error: kicad: It seems the App Suite source '/Applications/KiCad' is not there.
```

Following this error KiCad isn't installed but homebrew still consider it as installed so I have to reinstall it "manually" with `brew reinstall kicad`. I noticed that this error only happens when I update the software, the uninstall/reinstall/install always goes well, which is inconvenient because I can only reproduce the error when there is an update.

A particularity of this software is that it is installed inside a folder `KiCad` that also contains shortcuts for sub-softwares of KiCad suite. So I've the feeling that the problem is linked to the stanza `uninstall delete: [...]`, but I'm maybe wrong.

**Comment by CalaW** [comment]
Encountered the same issue when upgrading kicad from 6.0.5-0 to 6.0.6-1.

**Comment by pwhack** [comment]
I've had the same issue for many versions now and I've just futzed with it until I could get the updated version installed. I was drafting a bug report since you are able to reproduce the problem and I'm stumped on running `brew upgrade --force` (a checklist item for submitting a bug report) does get KiCad to upgrade. I don't think a `--force` should be needed for regular upgrades though so I'll submit a bug report and see if it gains traction.

https://github.com/Homebrew/homebrew-cask/issues/126832
